src/Uno.UWP/System/UserProfile/GlobalizationPreferences.cs

```diff
@@ -1,14 +1,57 @@
-﻿#if __ANDROID__
-using System;
+﻿using System;
 using System.Collections.Generic;
+using System.Runtime.InteropServices;
 using System.Text;
+
+#if __ANDROID__
 using Java.Util;
+#elif __IOS__
+using Foundation;
+#elif __SKIA__
+using Windows.WinRT;
+#endif
 
-namespace Windows.System.UserProfile
+namespace Windows.System.UserProfile;
+
+public static partial class GlobalizationPreferences
 {
-	public static partial class GlobalizationPreferences
+
+#if __ANDROID__ || __IOS__ || __SKIA__
+	public static IReadOnlyList<string> Languages =>
+#if __ANDROID__
+		new[] { Locale.Default.ToLanguageTag() };
+#elif __IOS__
+		NSLocale.PreferredLanguages;
+#elif __SKIA__
+		OperatingSystem.IsWindows() ? GetWinUserLanguageList() : throw new PlatformNotSupportedException();
```

Comment:
It's probably best to return an empty list, or maybe return the default thread culture.